docker/utils/utils.py

```diff
@@ -546,12 +546,6 @@ def datetime_to_timestamp(dt):
     return delta.seconds + delta.days * 24 * 3600
 
 
-def longint(n):
```

Comment:
all this does is allow users to enter their storage size in pounds (300lb) on python 2: